BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -38,6 +38,7 @@
   private LibraryModel libraryModel;
   private LibraryRecyclerViewAdapter adapter;
   private List<ShelfItem> selectedShelfItems;
+  private List<BookItem> selectedBookItems;
```

Comment:
Why do you create a selectedBookItems List in the LibraryFragment? In the library you can't select a book. You can only select shelves.